exercises/practice/leap/.approaches/introduction.md

```diff
@@ -1,10 +1,51 @@
 # Introduction
 
-There are various idiomatic approaches to solve Leap, such as
+There are various idiomatic approaches to solve Leap.
+All approaches listed below check for divisibility by 4, 100, and 400.
+However, they differ in the ways in which they combine these checks.
 
-- constructing [a logical expression][logical-expression],
-- using [guards][guards], and
-- using [a conditional expression][conditional-expression].
+
+## Approach: a logical expression
+
+```haskell
+isLeapYear :: Integer -> Bool
+isLeapYear year = divisibleBy 4 && (not (divisibleBy 100) || divisibleBy 400)
+  where
+    divisibleBy d = year `mod` d == 0
+```
+
+[Read more about this approach][logical-expression].
```

Comment:
Whilst it doesn't really make much sense here, something to consider for other approaches would be to add a one-liner before the  "Read more" sentence that describes the approach taken. But the approach is short enough I think to not need that.

Reply:
Yes, I do write these for other exercises.

I could only think of something for one of the three but not the other two, so I left it out.

Reply:
Perfect!